core/build-image.sh

```diff
@@ -107,6 +108,16 @@ buildah commit "${container}" "${repobase}/${reponame}"
 buildah rm "${container}"
 images+=("${repobase}/${reponame}")
 
+echo "Building the restic image..."
+container=$(buildah from alpine)
+reponame="restic"
+buildah run ${container} -- apk add restic
```

Comment:
```suggestion
buildah run ${container} -- apk add --no-cache restic
```

...or add cache index cleanup

```
f0657440c06d:/# du -h -s /var/cache/apk/
2.2M	/var/cache/apk/
```

Reply:
Cache is disabled inside the image

Reply:
2.2M of index files are retained anyway...